src/Model/Mail.php

```diff
@@ -170,7 +175,7 @@ public static function send($recipient = 0, $body = '', $subject = '', $replyto
 			$recip_host = substr($contact['url'], strpos($contact['url'], '://') + 3);
 			$recip_host = substr($recip_host, 0, strpos($recip_host, '/'));
 
-			$recip_handle = (($contact['addr']) ? $contact['addr'] : $contact['nick'] . '@' . $recip_host);
+			$recip_handle = $contact['addr'] ?: $contact['nick'] . '@' . $recip_host;
```

Comment:
This is some ancient code. We now always have got the contact's address in that field when the remote system does support this format. So the code here can be made slightly simpler.

BTW: The same is valid for the `$sender_handle`: This field is always filled and doesn't need to be constructed.

Reply:
Thanks for the heads-up!